applications/IgaApplication/custom_condition/penalty_coupling_condition.cpp

```diff
@@ -30,26 +30,40 @@ namespace Kratos
         const bool CalculateResidualVectorFlag)
     {
         KRATOS_TRY
+        const double penalty = GetProperties()[PENALTY_FACTOR];
 
         const auto& r_geometry_master = GetGeometry();
-        const auto& r_geometry_slave = r_geometry_master.GetSlaveGeometry();
+        const auto& r_geometry_slave = r_geometry_master.GetGeometryPart(1);
```

Comment:
You may define an enum in your master slave geometry to be more readable with something like:
```c++
const auto& r_geometry_slave = r_geometry_master.GetGeometryPart(MasterSlaveGeometry::SlavePart);
```

Reply:
But this does not has the possibility to have more than one slave. I will think about something.